spec/financial_spec.rb

```diff
@@ -81,6 +81,10 @@
 
       expect(results).to eq(70.14566694692749)
     end
+
+    it 'raises an exception when the payment amount is lower than the interest amount' do
+      expect { Exonio.nper(rate, -10, pv, fv, 1) }.to raise_error(ArgumentError)
```

Comment:
@cleicar Maybe to make it more explicit you could add the right message to the expectation? 

Something like this: 

```
expect { raise "oops" }.to raise_error(RuntimeError, "oops")
```